swarm/network/stream/delivery_test.go

```diff
@@ -446,6 +446,12 @@ func TestDeliveryFromNodes(t *testing.T) {
 	testDeliveryFromNodes(t, 2, dataChunkCount, false)
 	testDeliveryFromNodes(t, 4, dataChunkCount, true)
 	testDeliveryFromNodes(t, 4, dataChunkCount, false)
+
```

Comment:
this test really should not time out, it is either set to too small timeout duration or theres a bug.
we can investigate in  other issue

Reply:
I suspect we still have some resource leaks somewhere. As there are some tests which pass on their own, but get stuck if they run after many.

I vote for a separate issue.

Reply:
tracker: https://github.com/ethersphere/go-ethereum/issues/1247